docs/wiki/EL/evm.md

```diff
@@ -308,27 +308,44 @@ While Ethereum protocol undergoes many changes in each upgrade, changes in EVM a
 
 ### EVM
 
+The resources below has been effectively arranged based on your level of expertise.
+
+#### Level Zero: Beginner
```

Comment:
I like the idea of ordering these from beginner to advanced but I wouldn't necessarily call it levels
```suggestion
#### Beginner
```

Reply:
Sure makes sense.

lemme know if you have any suggestions, I will push changes accordingly.

Additionaly working on improving the WIKI section contents too , will create more PRs soon.

Reply:
Rest of your changes look good, I think  new resources are great!. I would just rename some of these titles and we can get it merged

Reply:
Sure, that sounds good.

Feel free to change titles as you see fit. Just lemme know if you need my help in changing anything. Thanks @taxmeifyoucan

Reply:
I just pushed a commit:

-  to fix titles, 
- removed "level" as rightly pointed out
- fixed some texts and punctuation as well.

Lemme know if any more changes are required in this branch. @taxmeifyoucan . Thanks.

Reply:
thanks, all looks good!